common/main/en.xml

```diff
@@ -256,7 +255,6 @@ annotations.
 			<language type="hak">Hakka Chinese</language>
 			<language type="haw">Hawaiian</language>
 			<language type="hax">Southern Haida</language>
-			<language type="hdn">Northern Haida</language>
```

Comment:
why?

Reply:
There was a hole whole where some deprecated language codes crept in (probably via raw XML editing or bulk upload).

Reply:
In CLDR, the aliased encompassed codes have always been classed as deprecated.